README.md

```diff
@@ -159,6 +159,9 @@ Though most configuration sources found online highly recommend running HA on a
 6. Paste `<device id>` of your smart plug and click submit.
 7. Repeat 4-6 times for each additional plug.
 
+### Google Integration
```

Comment:
Not sure why this got changed here, but this is what it should look like anyway.